vaccine_feed_ingest/runners/md/arcgis/normalize.py

```diff
@@ -0,0 +1,445 @@
+    "1729 Dual Highway Hagerstown Washington MD": "1729 Dual Hwy, Hagerstown, MD 21740",
+    "18726 N. Pointe Drive Hagerstown Washington MD": "18726 N Pointe Dr, Hagerstown, MD 21742",
+    "12154 Brittingham Lane, Princess Anne MD": "12154 Brittingham Ln, Princess Anne, MD 21853",
+    "3935 Erdman Ave #37 Baltimore MD": "3935 Erdman Ave #37, Baltimore, MD 21213",
+    # missing state, and according to walgreens' site, it's the wrong zip
+    "9621 Belair Rd Perry Hall 21128": "9621 Belair Rd Baltimore, MD 21236",
+    # missing space between state and zip
+    "1300 E North Ave Baltimore MD21213": "1300 E North Ave Baltimore MD 21213",
+}
+ADDRESS_RE = re.compile(
+    r"^(?P<street>.*),? (?P<city>[\w \-']+),?\s+(?P<state>md|maryland),?\s+(?P<zip>\d{5}(-\d{4})?)$",
+    flags=re.I,
+)
+
+
+def _get_address(site: dict) -> schema.Address:
+    raw = site["attributes"]["fulladdr"]
+    processed = SPECIAL_CASE_ADDRESSES.get(raw, raw).strip()
+    if (
+        site["attributes"]["name"] == "Leidos Field at Ripken Ironbirds Stadium"
+        and processed == "At this address"
+    ):
+        # extra paranoia for the "at this address" address
+        processed = "873 Long Dr, Aberdeen, MD 21001"
+    match = ADDRESS_RE.search(processed)
+    if not match:
+        raise RuntimeError(f"Unable to parse address string '{raw}'")
+    return schema.Address(
+        street1=match.group("street"),
+        city=match.group("city"),
+        state=schema.State.MARYLAND,
+        zip=match.group("zip"),
+    )
+
+
+def _get_availability(site: dict) -> Optional[schema.Availability]:
+    potentials = {
+        "Yes": schema.Availability(drop_in=True),
+        "No": schema.Availability(drop_in=False),
+    }
+    return potentials.get(site["attributes"]["WalkUpAvailable"])
+
+
+def _get_id(site: dict) -> str:
+    # Could parse these from directory traversal, but do not for now to avoid
+    # accidental mutation.
+    runner = "md"
+    site_name = "arcgis"
+
+    service_item = site["attributes"]["service_item_id"]
+    layer = site["attributes"]["layer_id"]
+    data_id = site["attributes"]["OBJECTID"]
+
+    return f"{runner}_{site_name}:{service_item}_{layer}_{data_id}"
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = []
+    potentials = [
+        # (attribute, Contact field)
+        ("PreRegistrationURL", "website"),
+        ("schedule_url", "website"),
+        ("scheduling_contact_phone", "phone"),
+        ("website_url", "website"),
+    ]
+    for attr, field in potentials:
+        if site["attributes"][attr]:
+            try:
+                contacts.append(schema.Contact(**{field: site["attributes"][attr]}))
+            except ValidationError:
+                logger.exception(
+                    f"Validation error adding Contact '{attr}' = '{site['attributes'][attr]}', skipping"
+                )
```

Comment:
It looks like this logging currently outputs >1.5K lines while normalizing `md/arcgis` (because of the Traceback).

Could you shift this to something like `logger.info` in order to:
* not log the Traceback
* properly categorize this as informational, not errors that need to be addressed

Alternatively - if we're relying on `try`/`catch` for normalization here - maybe we shouldn't log _at all_?

Reply:
Yup, not a problem.